packages/components/src/schemas/main.ts

```diff
@@ -0,0 +1,36 @@
+import type { TSchema } from "@sinclair/typebox"
+
+import type { IsomerComponentTypes } from "~/types"
+import { IsomerPageSchema } from "~/types"
+import {
+  IsomerComplexComponentsMap,
+  IsomerNativeComponentsMap,
+} from "./components"
+
+const definitions = {
+  components: {
+    complex: IsomerComplexComponentsMap,
+    native: IsomerNativeComponentsMap,
+  },
+}
+
+export const schema: TSchema = {
+  $schema: "http://json-schema.org/draft-07/schema#",
+  title: "Isomer Next Page JSON",
```

Comment:
Page Schema*

Reply:
Fixed in d819f9d5e2f3211a6951bde3dab63903ec01b7dc.